src/tests/transform-result.spec.ts

```diff
@@ -1134,3 +1168,197 @@ describe('transformComment', (): void => {
+            {
+                done: false,
+                incomplete: true,
+                value: [],
+                variant: Variant.FunctionAlias
+            },
+            {
+                hit: 0,
+                done: false,
+                index: 0,
+                name: '',
+                variant: Variant.FunctionAlias
+            }
+        ],
+        [
+            {
+                done: true,
+                incomplete: false,
+                value: ['1'],
+                variant: Variant.FunctionAlias
+            },
+            {
+                hit: 0,
+                done: true,
+                index: 0,
+                name: '',
+                variant: Variant.FunctionAlias
+            }
+        ],
+        [
+            {
+                done: true,
+                incomplete: true,
+                value: ['1', '10'],
+                variant: Variant.FunctionAlias
+            },
+            {
+                hit: 0,
+                done: true,
+                index: 0,
+                name: '',
+                variant: Variant.FunctionAlias
+            }
+        ],
+        [
+            {
+                done: true,
+                incomplete: true,
+                value: ['1', '10', 'test'],
+                variant: Variant.FunctionAlias
+            },
+            {
+                hit: 10,
+                done: true,
+                index: 1,
+                name: 'test',
+                variant: Variant.FunctionAlias
+            }
+        ],
+        [
+            {
+                done: false,
+                incomplete: false,
+                value: ['NaN', 'NaN', 'n'],
+                variant: Variant.FunctionAlias
+            },
+            {
+                hit: 0,
+                done: false,
+                index: 0,
+                name: 'n',
+                variant: Variant.FunctionAlias
+            }
+        ],
+        [
+            {
+                done: true,
+                incomplete: true,
+                value: ['2', '11', 'shouldBeFine', 'this', 'should', 'not', 'be', 'used'],
+                variant: Variant.FunctionAlias
+            },
+            {
+                hit: 11,
+                done: true,
+                index: 2,
+                name: 'shouldBeFine',
+                variant: Variant.FunctionAlias
+            }
+        ]
+    ]
+
+    processTestData(testData, transformFunctionAlias)
+})
+
+describe('transformFunctionLeader', (): void => {
+    const testData: TestData<Variant.FunctionLeader, FunctionLeaderEntry> = [
+        [
+            {
+                done: false,
+                incomplete: false,
+                value: null,
+                variant: Variant.FunctionLeader
+            },
+            {
+                done: false,
+                index: 0,
+                lineEnd: 0,
+                lineStart: 0,
+                variant: Variant.FunctionLeader
+            }
+        ],
```

Comment:
`value` can (or rather _should_) never be `null`, so having a test for it seems unnecessary.